### ISSUE TITLE
Query.substances errors when `limit` argument is too high

### ISSUE THREAD

**Issue by dlbnco** [psychonautwiki/bifrost#25]
**Query**
```graphql
query getSubstances($limit: Int, $offset: Int) {
  substances(limit: $limit, offset: $offset) {
    name
    class {
      psychoactive
      chemical
    }
    featured
  }
}
```
**Variables**

Seems to error when values higher than 51 are used.

```json
{
  "limit": 100
}
```
**Response**
```json
{
  "data": {
    "substances": null
  },
  "errors": [
    {
      "locations": [
        {
          "column": 45,
          "line": 1
        }
      ],
      "message": "Cannot destructure property `query` of 'undefined' or 'null'.",
      "path": [
        "substances"
      ]
    }
  ]
}
```

**Comment by 19h** [comment]
I suspect values higher than 50 cause a hard-error on the internal API level and this bubbles up and the bad value cannot be returned because it violates the GraphQL schema. I'll take a look at it soon.

**Comment by predme** [comment]
Any update on this? My app has been unable to query the API for a while now.

**Comment by 19h** [comment]
@predme I'm not sure why this is a critical error. Please elaborate why you need to query for more than 50 items via the API.

**Comment by predme** [comment]
@19h My app queries the API for all the substances, caches it and displays the result in the app. To query all substances, I set the limit to 9999. Unfortunately, if it's the first time the user starts the app, I won't be able to cache the result, nor give any substance info to the user.

**Comment by 19h** [comment]
Thanks for your explanation. I will treat this as a critical error.

**Comment by 19h** [comment]
A short investigation showed this is not actually related to pagination. I will close this issue and will track the status of this breakage in #26.

**Comment by 19h** [comment]
@dlbnco @predme fixed in #26.
